### ISSUE TITLE
README should point to new post-re-organisation JSON locations

### ISSUE THREAD

**Issue by petertseng** [exercism/x-common#372]
README currently has:

```Each problem can have a file <problem>.json```

Rendered, this just shows up as "Each problem can have a file .json", but that's a separate issue.

The more important thing is that this should get changed to match the location after #351 moved to per-exercise directories.

**Comment by kytrinyx** [comment]
Good catch, fixing that now.
